src/lib/components/ProductWidget/ProductWidgetVariation0.svelte

```diff
@@ -56,7 +56,7 @@
 				{product.shortDescription}
 			</p>
 		</a>
-		{#if canAddToCart}
+		{#if canAddToCart && !product.hasSellDisclaimer}
```

Comment:
Rather than modifying all those variations, can you change `canAddToCart`?

```
	$: canAddToCart =
		canBuy && (!product.availableDate || product.availableDate <= new Date() || !!product.preorder);
```